src/services/ThemeContext/ThemeContext.tsx

```diff
@@ -0,0 +1,88 @@
+import React, {createContext, useContext, useState, ReactNode , FC} from "react"
+import {HaskiTheme, DarkTheme, AltTheme} from '@common/utils'
+import {ThemeProvider} from "@common/theme";
+
+import {postUserSettings} from "../Theme/postUserSettings";
```

Comment:
alle imports noch in die dazugehörigen index packen, sodass man von '@..' importieren kann.